src/Symfony/Bundle/FrameworkBundle/Command/AbstractConfigCommand.php

```diff
@@ -52,6 +52,9 @@ protected function listBundles($output)
         }
     }
 
+    /**
+     * @return ExtensionInterface
```

Comment:
that's for lower branches (all docblock changes on PRs on master should be on lower branches, ping @symfony/deciders and all other reviewers).

Reply:
see #32382

Reply:
Thanks @fabpot